exercises/practice/largest-series-product/.meta/example.jl

```diff
@@ -0,0 +1,16 @@
+function largest_product(s::AbstractString, span::Int)
```

Comment:
```suggestion
function largest_product(s, span)
```

Type annotation should mainly be used for dispatch, which isn't the case here. It's often a good idea to leave them as general as possible, so the examples on exercism afair mostly don't specify the type.